src/Connection/Aggregate/RedisCluster.php

```diff
@@ -276,11 +280,11 @@ protected function guessNode($slot)
      */
     protected function createConnection($connectionID)
     {
-        $separator = strrpos($connectionID, ':');
+        list($host,$port) = explode(':', $connectionID, 2);
```

Comment:
This change breaks some stuff as it reverts a fix purposely shipped in a7ee807 that aimed to make Predis as a whole working with IPv6 addresses, see #239 for more details.